ecosystem/typescript/sdk/src/types/bip39-light.d.ts

```diff
@@ -0,0 +1,13 @@
+declare module 'bip39-light' {
```

Comment:
Why do you have to add the types manually? It could not be installed as a dependency?

Reply:
`@types/bip39-light` is not exist. but I'll replace to `scure-bip39` and remove this.